.circleci/config.yml

```diff
@@ -94,8 +106,9 @@ workflows:
       - ruby_2_7_rails_5_2
       - ruby_2_7_rails_6_0
       - ruby_2_7_rails_6_1
+      - ruby_2_7_rails_7_0
       - ruby_2_7_rails_main
-      - ruby_3_0_rails_6_1
```

Comment:
If you don't need `ruby_3_0_rails_6_1`  job definition, it may be better to delete (line 80)

Reply:
Thanks! Fix it https://github.com/kufu/activerecord-bitemporal/pull/85/commits/9fe1d1561d77aaa86387e58f3839ccb85df0ec75